src/components/ButtonPanel.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+class ButtonPannel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <p>
+          ButtonPannel component connected
+        </p>
+        <div className="" id="G1">
+          <button type="submit">AC</button>
+          <button type="submit">+/-</button>
+          <button type="submit">%</button>
+          <button type="submit">+</button>
+        </div>
+        <div className="" id="G2">
+          <button type="submit">7</button>
```

Comment:
You should use the `Button` component here, instead of using the `button` element by passing the button name as `name` props. For example:

```jsx
// This should be used, assuming we have imported Button component and it has a name prop
<Button name="AC" />

// Instead of this
<button type="submit">AC</button>
```